test/activate-event.html

```diff
@@ -49,8 +49,13 @@
 
       var s;
 
-      setup(function () {
+      setup(function(done) {
         s = fixture('test');
+
+        s.addEventListener('iron-items-changed', function onIronItemsChanged() {
```

Comment:
Seems good. The main reason I ask is because we asked a lot of users to use the `forceSynchronousItemUpdate` as a fix to making iron-selector asynchronous which caused an overwhelming amount of blowback from our users.